jib-maven-plugin/README.md

```diff
@@ -220,7 +220,8 @@ Field | Type | Default | Description
 `to` | [`to`](#to-object) | *Required* | Configures the target image to build your application to.
 `from` | [`from`](#from-object) | See [`from`](#from-object) | Configures the base image to build your application on top of.
 `container` | [`container`](#container-object) | See [`container`](#container-object) | Configures the container that is run from your image.
-`extraDirectory` | [`extraDirectory`](#extradirectory-object) / string | `(project-dir)/src/main/jib` | Configures the directory used to add arbitrary files to the image.
+`extraDirectory` | `extraDirectory` / string | `(project-dir)/src/main/jib` | Deprecated. Use `extraDirectories`.
```

Comment:
I think we can just remove this, we don't want people to use it so it just takes up space.